src/oasis-nexus/api.ts

```diff
@@ -416,7 +427,7 @@ export const useGetRuntimeAccountsAddress: typeof generated.useGetRuntimeAccount
           if (status !== 200) return data
           return groupAccountTokenBalances({
             ...data,
-            address_eth: getEthAddressForAccount(data, address),
+            address_eth: address.eth ?? getEthAddressForAccount(data, address.oasis),
```

Comment:
getEthAddressForAccount was supposed to already be doing this, if second parameter is eth address
fixed in https://github.com/oasisprotocol/explorer/pull/1039
previous issue https://github.com/oasisprotocol/explorer/issues/1037